test/mock_sdk.dart

```diff
@@ -164,7 +164,8 @@ library dart.async;
 
 import 'dart:math';
 
-part 'stream.dart';
+class Stream<T> {}
+abstract class StreamTransformer<S, T> {}
```

Comment:
@pq @bwilkerson 

this fixed it, on windows.

Any complaints?

Any insights into the underlying problem? I didn't see any windows paths or anything that would cause it, and, strangely, Stream works _unless the analyzer is creating a Stream to compare against the return type of an `async*` function_...

Reply:
> Any insights into the underlying problem?

Not off-hand. I'd have to try debugging it to see what's going on.

Reply:
I'm comfortable landing this as is.  Maybe open a tracking issue to capture context if we loop back to further investigate?